Gems/ROS2/Code/Include/ROS2/ROS2Bus.h

```diff
@@ -33,6 +34,12 @@ namespace ROS2
         //! @note Alternatively, you can use your own node along with an executor.
         virtual std::shared_ptr<rclcpp::Node> GetNode() const = 0;
 
+        //! Attach handler to ROS2 node change events.
+        //! You can use this method to correctly initialize SystemComponents that depend on ROS2Node.
+        //! @note callback is active as long as handler is not destroyed.
+        using NodeChanged = AZ::Event<std::shared_ptr<rclcpp::Node>>;
```

Comment:
You can consider placing alias at the top of class definition - this is easier to read than placing it between doc and method signature.
+
@param is missing in documentation for this method.

Reply:
I would optionally consider adding "...Event" to this alias name, if current form is not the part of convention in O3DE.